pkg/scan/scan.go

```diff
@@ -153,6 +154,34 @@ func renderDigest(manifest payloadGetter) (digest.Digest, error) {
 	return dig, nil
 }
 
+func handleManifestLists(reg types.Registry, remote, ref string, manifests []manifestlist.ManifestDescriptor) (digest.Digest, []string, error) {
+	if len(manifests) == 0 {
+		return "", nil, errors.Errorf("no valid manifests found for %s:%s", remote, ref)
+	}
+	if len(manifests) == 1 {
```

Comment:
is this case needed? should be covered by the loop, right? plus, I think this allows non-linux platforms, too

Reply:
So this actually deviates from what central does today:
```
	if len(manifestList.Manifests) == 1 {
		return handleManifests(r, manifestList.Manifests[0].MediaType, remote, manifestList.Manifests[0].Digest.String())
	}
	```